tests/validation/tests/v3_api/common.py

```diff
@@ -31,6 +33,19 @@
     os.path.dirname(os.path.realpath(__file__)),
     "rancher_env.config")
 
+is_windows_os = TEST_OS == "windows"
+if is_windows_os:
+    DEFAULT_TIMEOUT = 300
+skip_test_windows_os = pytest.mark.skipif(
+    is_windows_os,
+    reason='Tests Skipped for including Windows nodes cluster')
+
+
+def ensure_workload_scheduling():
+    if is_windows_os:
+        return {"node": {"requireAll": ["beta.kubernetes.io/os = windows"]}}
+    return {}
```

Comment:
Maybe we need to add linux scheduling rule too.

Reply:
Updated:
```
def ensure_workload_scheduling(os_type=TEST_OS):
    if not os_type:
        return {}
    return {"node": {"requireAll": ['beta.kubernetes.io/os = {0}'.format(os_type)]}}
```